Code/Tools/FBuild/FBuildCore/Helpers/VSProjectGenerator.cpp

```diff
@@ -86,6 +88,48 @@ void VSProjectGenerator::AddFiles( const Array< AString > & files )
     guid.Format( "{%08x-6c94-4f93-bc2a-7f5284b7d434}", CRC32::Calc( projectNameNormalized ) );
 }
 
+// IsFilenameAHeaderFile
+//------------------------------------------------------------------------------
+static inline const bool IsFilenameAHeaderFile( const AString & fileName )
+{
+    // TODO: Add support for other header types?
+    return ( fileName.EndsWith( 'h' ) || fileName.EndsWith( ".hpp" ) || fileName.EndsWith( ".hxx" ) || fileName.EndsWith( ".inl" ) );
```

Comment:
These are all case sensitive - I think they need to be case insensitive.

Reply:
Another good catch - would that be `EndsWithI`? I think I wrote this code in a bit too much haste given that I'm not an expert at the codebase :/

Reply:
Yeah, EndsWithI :)